c2rust-analyze/src/main.rs

```diff
@@ -425,6 +426,8 @@ fn run(tcx: TyCtxt) {
         gacx.fn_sigs.insert(ldid.to_def_id(), lsig);
     }
 
+    gather_foreign_sigs(&mut gacx, tcx);
```

Comment:
In [this comment](https://github.com/immunant/c2rust/pull/980/files#r1255886959) this is where I was wondering if we already have what you need without needing to filter all fns